selfdrive/boardd/pandad.py

```diff
@@ -27,6 +28,7 @@ def flash_panda(panda_serial: str) -> Panda:
   panda = Panda(panda_serial)
 
   fw_signature = get_expected_signature(panda)
+  internal_panda = panda.is_internal() and not panda.bootstub
```

Comment:
how do you recover an internal panda using gpio if it's already stuck in bootstub on startup?

Reply:
not handled in this PR since the bootstub doesn't implement the endpoint to query hw type. I'm thinking we can handle this nicely by adding internal panda types to the HW abstraction layer and wait until all internal pandas are up, but want to keep this PR simple